src/commands/run.rs

```diff
@@ -42,38 +43,211 @@ Configuration is loaded from the following sources (highest precedence first):
+    /// filters. See the link below for more details.
+    ///
+    /// No filter by default.
+    ///
+    /// https://docs.rs/tracing-subscriber/0.2.17/tracing_subscriber/filter/struct.EnvFilter.html
+    #[structopt(long = "--traces-filter", env = "TRACES_FILTER")]
+    pub traces_filter: Option<String>,
+
+    /// Tracing: sampler type
+    ///
+    /// Can be one of:
+    /// always_on, always_off, traceidratio,
+    /// parentbased_always_on, parentbased_always_off, parentbased_traceidratio
+    ///
+    /// https://github.com/open-telemetry/opentelemetry-specification/blob/v1.1.0/specification/sdk-environment-variables.md#general-sdk-configuration
+    #[structopt(
+        long = "--traces-sampler",
+        env = "TRACES_SAMPLER",
+        default_value = "parentbased_traceidratio"
+    )]
+    pub traces_sampler: tracing::TracesSampler,
+
+    #[rustfmt::skip]
+    /// Tracing: sampler argument
+    ///
+    /// Valid range: [0.0, 1.0].
+    /// 
+    /// Only used if OTEL_TRACES_SAMPLER is set to
+    /// parentbased_traceidratio (default) or traceidratio.
+    ///
+    /// With sample parentbased_traceidratio, the following rules apply:
+    /// - if parent is sampled, then all of its children are sampled
+    /// - else sample this portion of traces (0.5 = 50%)
+    ///
+    /// https://github.com/open-telemetry/opentelemetry-specification/blob/v1.1.0/specification/sdk-environment-variables.md#general-sdk-configuration
+    #[structopt(
+        long = "--traces-sampler-arg",
+        env = "TRACES_SAMPLER_ARG",
+        default_value = "1.0",
+        verbatim_doc_comment
+    )]
+    pub traces_sampler_arg: f64,
+
+    /// Tracing: OTLP (eg OpenTelemetry collector) network hostname
+    ///
+    /// Only used if TRACES_EXPORTER is "otlp".
+    ///
+    /// Protocol is gRPC. HTTP is not supported.
+    #[structopt(
+        long = "--traces-exporter-otlp-host",
+        env = "TRACES_EXPORTER_OTLP_HOST",
+        default_value = "localhost"
+    )]
+    pub traces_exporter_otlp_host: String,
+
+    /// Tracing: OTLP (eg OpenTelemetry collector) network port
+    ///
+    /// Only used if TRACES_EXPORTER is "otlp".
+    ///
+    /// Protocol is gRPC. HTTP is not supported.
+    #[structopt(
+        long = "--traces-exporter-otlp-port",
+        env = "TRACES_EXPORTER_OTLP_PORT",
+        default_value = "4317"
+    )]
+    pub traces_exporter_otlp_port: NonZeroU16,
+
+    /// Tracing: Jaeger agent network hostname
+    ///
+    /// Protocol is Thrift/Compact over UDP.
+    ///
+    /// Only used if TRACES_EXPORTER is "jaeger".
+    #[structopt(
+        long = "--traces-exporter-jaeger-agent-host",
+        env = "TRACES_EXPORTER_JAEGER_AGENT_HOST",
+        default_value = "0.0.0.0"
+    )]
+    pub traces_exporter_jaeger_agent_host: String,
+
+    /// Tracing: Jaeger agent network port
+    ///
+    /// Protocol is Thrift/Compact over UDP.
+    ///
+    /// Only used if TRACES_EXPORTER is "jaeger".
```

Comment:
Same comment here, should this be:

```suggestion
    /// Only used if `--traces-exporter` is "jaeger".
```